packages/core/src/messages/responses/enums/store/scalar/index.ts

```diff
@@ -0,0 +1,21 @@
+export enum StorageGetResponse {
+  Success = 'Success',
+  Error = 'Error',
+}
+
+export enum StorageItemType {
```

Comment:
have we aligned on a name for this?
this is called `StorageValueType` in the Go PR

Reply:
@malandis `StorageItemType` vs `StorageValueType`?